app/types/graphql.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Hasura aggregate data object
+ *
+ * This is loosely typed and does not exhaustively cover all supported aggregation fields
+ * See: https://hasura.io/docs/2.0/api-reference/graphql-api/query/#aggregateobject
+ */
+export interface HasuraAggregateData {
+  aggregate: {
+    count?: number;
+    sum: Record<string, number>;
+    avg: Record<string, number>;
+    max: Record<string, number>;
+    min: Record<string, number>;
+  };
+}
```

Comment:
We're only using `count` at the moment, but i went ahead and added these common aggregates 🤷